packages/kbn-optimizer/limits.yml

```diff
@@ -109,7 +109,7 @@ pageLoadAssetSize:
   presentationUtil: 58834
   profiling: 36694
   remoteClusters: 51327
-  reporting: 57003
+  reporting: 288461
```

Comment:
Can/did we look into loading these assets asynchronously?

Reply:
I got it down to 44274 by refactoring the imports from kbn/reporting-common and kbn/reporting-export-types-helpers for the public and common parts of the reporting plugin 👍🏻

Reply:
I have separated more config-schema code from the public side of Reporting, and as of now the limits.yml file is no longer changed in this PR.